joomla/joomla-cms
# Single registered article guest view

Pull Request for Issue #9830
#### Testing Instructions

Create a single article with an intro and full image and a readmore
Set the article to registered only
Create a menu link to the article of type single article
In the menu options make sure show unauthorised links is set to show

Now check the menu item on the front end - you should see the introtext but not the intro image and when you login you see all the text and the full image

Apply the patch
Now check the menu item on the front end - you should see the introtext and the intro image and when you login you see all the text and the full image
